tools/pv_tree_generator/build_tree.py

```diff
@@ -46,6 +47,8 @@ def build_tree_recursive(pos, level, pop_obs_spec, stat_vars, show_all,
         'sv_set': set(),
         'search_count': 0,
         'search_sv_set': set(),
+        'measuredProperty': prop_ui_node.mprop,
+        'placeTypes': PLACETYPES,
```

Comment:
Can you check original code, there is a logic to compute the places available for each stats var instead of using all place types

Reply:
added the logic, need place info of statvars